test/services/ConsignmentStatusServiceSpec.scala

```diff
@@ -32,12 +32,12 @@ class ConsignmentStatusServiceSpec extends WordSpec with Matchers with MockitoSu
 
   "consignmentStatus function" should {
     "return the consignment status" in {
-      val data = Option(gcs.Data(Option(gcs.GetConsignment(CurrentStatus(Option("TestStatus"))))))
+      val data = Option(gcs.Data(Option(gcs.GetConsignment(CurrentStatus(Option("TestStatus"), None)))))
       val response = GraphQlResponse(data, Nil)
       when(getConsignmentStatusClient.getResult(token, gcs.document, Some(gcs.Variables(consignmentId))))
         .thenReturn(Future.successful(response))
       val status = consignmentStatusService.consignmentStatus(consignmentId, token).futureValue
-      status should equal(Option(CurrentStatus(Option("TestStatus"))))
+      status should equal(Option(CurrentStatus(Option("TestStatus"), None)))
```

Comment:
Could this use a real value rather than `None`, to make sure that the real value has been passed through rather than a default?